roles/sft-server/templates/nginx.service.override.j2

```diff
@@ -1,4 +1,9 @@
 [Service]
 LimitNOFILE={{ nginx_rlimit_nofile }}
+
 ExecStartPre=/usr/sbin/nginx -t
 ExecReload=/usr/sbin/nginx -t
```

Comment:
the reload command seems incorrect, by the way. That just tests the configuration.

Reply:
Interestingly not, which I just learned myself. By default, these systemd directives are handled additive. That means in our case, the one directive configured during installation: `ExecReload=/bin/kill -s HUP $MAINPID` was executed and afterwards the test (`-t`).  Of course, this is not the desired behaviour, which is why I did it [properly](https://github.com/wireapp/ansible-sft/pull/18).